dotnet/xml/_filter.xml

```diff
@@ -1,8 +1,170 @@
 <?xml version="1.0" encoding="utf-8"?>
 <filter>
   <apiFilter apply="true">
+    <namespaceFilter name="Microsoft.EntityFrameworkCore.ChangeTracking.Internal">
```

Comment:
Would this list be, all namespace across all versions which should be filtered out?

Reply:
yes

Reply:
Yes.